virttest/utils_conn.py

```diff
@@ -1292,17 +1300,18 @@ def server_setup(self, on_local=False):
             raise ConnMkdirError(self.libvirt_pki_private_dir, output)
 
         if scp_new_cacert == 'no':
-            scp_dict = {servercert_path: self.libvirt_pki_dir,
-                        serverkey_path: self.libvirt_pki_private_dir}
+            params_scp = {'serverkey': serverkey_path,
+                          'servercert': servercert_path}
         else:
-            scp_dict = {cacert_path: self.pki_CA_dir,
-                        cakey_path: self.pki_CA_dir,
-                        servercert_path: self.libvirt_pki_dir,
-                        serverkey_path: self.libvirt_pki_private_dir}
             cmd = "mkdir -p {0}".format(self.pki_CA_dir)
             status, output = server_session.cmd_status_output(cmd)
             if status:
                 raise ConnMkdirError(self.pki_CA_dir, output)
+            params_scp = {'cacert': cacert_path,
+                          'cakey': cakey_path,
+                          'serverkey': serverkey_path,
+                          'servercert': servercert_path}
+        scp_dict = self._update_scp_list(self.scp_list_server, params_scp)
```

Comment:
I notice that previous implementation don't this step. Is the step conditionally required or necessary required now?

Reply:
If you are asking to invoke for _update_scp_list(), then this step is necessary for all. It is a new created function which applies to all. If the user does not provide scp_list_client/scp_list_server, then invoking  this function makes no effect on them. It is just unified entry.